### ISSUE TITLE
Add Captioning Guidelines

### ISSUE THREAD

**Issue by francisrupert** [USDA-FSA/fsa-design-system#70]
- [x] Drafting
- [x] Drafted

**Comment by brianmillsUX** [comment]
Should be all set, per guides/accessibility/multimedia as follows: "Closed captioning - Videos with audio require synchronized closed captioning. Anything said in the video must be included in the closed captioning, and anything in the closed captioning must be said in the video."
